assets/src/components/PreviewBanner/PreviewBanner.lazy.js

```diff
@@ -0,0 +1,11 @@
+import React, { lazy, Suspense } from 'react';
+
+const LazyPreviewBanner = lazy(() => import('./PreviewBanner'));
+
+const PreviewBanner = props => (
+  <Suspense fallback={null}>
+    <LazyPreviewBanner {...props} />
+  </Suspense>
+);
```

Comment:
You're generating these through some command, correct? Are you using it here? It doesn't look like it to me. Would this lazy approach make more sense for components with state? I need to be educated on how these work, when they make sense. Creating more directories and files for this (especially if we're not going to use it) seems like potentially too much.

Reply:
Yeah I am generating them with a command.  I was operating under the assumption that the tool was implementing best practices.  I have tried to use the lazy version of the component but it looks like it requires some additional configuration to make it work.  I had left the previous one in b/c I thought it might make sense to take a pass at a later date to use the lazy version (as I assumed it was preferred for performance)  I didn't think there was really any downside to having a file that wasn't used but I can delete it.

Reply:
I guess I need to learn more about these. I generally prefer to keep the codebase compact and only including code you're actually going to run. That might be a narrow-minded approach though.

Reply:
As do I.  I don't think it would be difficult to come up with it again if it was gone, however if it was a practice to use eventually I think the folder structure makes sense.  I'll go ahead and delete it for now.  Not like lazy loading something this small would be likely to be impactful anyways, I imagine.